packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -65,6 +66,8 @@ class DebuggerScreenBody extends StatefulWidget {
 
   static final codeViewKey = GlobalKey(debugLabel: 'codeViewKey');
   static final scriptViewKey = GlobalKey(debugLabel: 'scriptViewKey');
+  static const copyToClipboardButtonKey =
```

Comment:
rename to callStackCopyButtonKey since there are other places on the debugger screen that have a copy control. this will help us keep those straight.